Dockerfile

```diff
@@ -33,7 +33,8 @@ RUN set -x \
  && /fossology/utils/fo-installdeps -e -y \
  && $_cleanup
 RUN curl -sS https://getcomposer.org/installer | php && \
-    mv composer.phar /usr/local/bin/composer
+	mkdir /tmp/composer && \
+    mv composer.phar /tmp/composer/composer
```

Comment:
If one calls the following
```bash
$ composer_download
$ sudo composer_install
```
one is no longer able to execute `composer_download` again, since root owns some of the files in `~/.composer`. Thus loosing the binary at `/tmp/composer/composer` is hard problem. (tested on vagrant setup)